packages/js/product-editor/src/style.scss

```diff
@@ -1,14 +1,15 @@
-@import 'components/editor/style.scss';
-@import 'components/product-section-layout/style.scss';
-@import 'components/edit-product-link-modal/style.scss';
-@import 'components/details-categories-field/style.scss';
-@import 'components/details-categories-field/create-category-modal.scss';
-@import 'components/header/style.scss';
-@import 'components/images/editor.scss';
-@import 'components/block-editor/style.scss';
-@import 'components/section/editor.scss';
-@import 'components/tab/editor.scss';
-@import 'components/tabs/style.scss';
-@import 'components/details-summary-block/style.scss';
-@import 'components/product-mvp-ces-footer/style.scss';
-@import 'components/product-mvp-feedback-modal/style.scss';
+@import "components/editor/style.scss";
```

Comment:
Any reason to update double quotes?  No opinion here just noting that git conflicts will occur because of this.

Reply:
Ohh!! just a typo here thanks.

Reply:
Done -> https://github.com/woocommerce/woocommerce/pull/37567/commits/30c1144587eb8271f1b3939dc95472d0d639929c